docs/features/back-up-and-restore.mdx

```diff
@@ -5,31 +5,99 @@ description: Learn how to back up and restore your Pieces Database without losin
 
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
-import Admonition from '@theme/Admonition';
+import Video from "/src/components/Video";
+import Admonition from "@theme/Admonition";
 
 # Backup your Pieces Database
 
+Backing up your Pieces Database is a crucial step to ensure you don’t lose any of your code snippets. This guide will walk you through how to easily back up and restore your Pieces Database within the Pieces Desktop App.
```

Comment:
Nice - might want a small note in here around best time to use or use cases where this is helpful 

i.e. you back up from pieces when you're done working for the day on your personal computer and then restore it when you get on your work device.. and visa versa i.e. when your done with your work device, you back it up, and then you restore your personal device from your last backup

Reply:
updated